migrations/20241212130340-test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    await queryInterface.addColumn('notification', 'created_at', {
```

Comment:
Where is deleting create_at?
Main feature of migrations, is that u creating new empty database, then running all the migrations and the result tables content is the same as in other developers.
If u will run ur migrations on empty datatbase u will catch an error, because first migration creating created_at column, and the second one creating it again.
So second migration must contains code to delete and then recreate this column